packages/notebook-extension/src/index.ts

```diff
@@ -1777,7 +1778,8 @@ function activateNotebookCompleterService(
     const completerContext = {
       editor: notebook.content.activeCell?.editor ?? null,
       session: notebook.sessionContext.session,
-      widget: notebook
+      widget: notebook,
+      sanitizer: new Sanitizer()
```

Comment:
Can we hoist the constructor here (and similar cases) a level higher so that we do not create a new instance of sanitizer on each `updateCompleter()` (etc) call? I think that only one instance of the Sanitizer should suffice for the entire notebook-extension, one instance for fileeditor-extension, and the same for console.

Reply:
Sure thing sorry about that. Bad practise. I'll update it.